cpp/src/parquet/properties.h

```diff
@@ -275,10 +282,33 @@ class PARQUET_EXPORT WriterProperties {
           page_checksum_enabled_(properties.page_checksum_enabled()),
           size_statistics_level_(properties.size_statistics_level()),
           sorting_columns_(properties.sorting_columns()),
-          default_column_properties_(properties.default_column_properties()) {}
+          default_column_properties_(properties.default_column_properties()),
+          cdc_enabled_(properties.cdc_enabled()),
+          cdc_size_range_(properties.cdc_size_range()),
+          cdc_norm_factor_(properties.cdc_norm_factor()) {}
 
     virtual ~Builder() {}
 
+    Builder* enable_cdc() {
```

Comment:
What about using its full name instead of `cdc`? It may not be obvious to people who are not familiar with it and be mixed with another norm `change data capture`.

Reply:
Agree, it is confusing with change data capture, going to spell out.

Reply:
I spelled it out everywhere. It is a little bit verbose, but definitely less confusing.